pylint/pyreverse/utils.py

```diff
@@ -155,7 +149,11 @@ def walk(self, node, _done=None):
         self.leave(node)
         assert node.parent is not node
 
-    def get_callbacks(self, node):
+    def get_callbacks(
+        self, node: nodes.NodeNG
+    ) -> tuple[
+        Callable[[nodes.NodeNG], Any] | None, Callable[[nodes.NodeNG], Any] | None
+    ]:
```

Comment:
There must be a better way to make a shorthand notation for it (maybe by creating a `TypeVar`?), but I could not come up with it.
Maybe one of the typing pros @DanielNoord or @cdce8p can help?

Reply:
You can use:
`MyRecurringType = Callable[[int, int, int, int, int, int, ...], float]` at the top of the file just below the imports and then re-use it throughout the file.
The only thing you need to look out for is that `Callable` from `collections.abc` then needs to be imported from `typing`. As this way `MyRecurringType` is defined at runtime and on <3.8 `Callable` isn't subscriptable at runtime.

Reply:
For the callable parameter type, take a look at my comment here as it's not completely correct (but probably the most practical):
https://github.com/PyCQA/pylint/blob/8df920b341a872b85754c910598d27a8846988f9/pylint/utils/ast_walker.py#L18-L24

--
As a side note: Shouldn't the return type be `None` instead of `Any`?